src/RemoteAPI/class-analytics-posts-api.php

```diff
@@ -79,4 +78,17 @@ class Analytics_Posts_API extends Remote_API_Base {
 	public function get_posts_analytics( $api_params ) {
 		return $this->get_items( $api_params, true ); // @phpstan-ignore-line
 	}
+
+	/**
+	 * Sets the request's options for the remote API call.
+	 *
+	 * @since 3.9.0
+	 *
+	 * @return array<string, mixed> The array of options.
+	 */
+	protected function set_request_options(): array {
```

Comment:
The name here should be `get_request_options`

Reply:
Done in https://github.com/Parsely/wp-parsely/pull/1789/commits/9be6751372f8c6e282dd1d19ee5942d313845c8c